### PR TITLE
User status broadcast complete

### PR DESCRIPTION
**Pull request by Matthew1231A**
One of the features we wanted to add is an alert when a user logs on/off. This code broadcasts to all connected clients the name of the user who is changing their state.

### CODE REVIEW

**Comment by andjf** [comment]
🤡

File: src/main.rs [hunk 1]
```diff
@@ -26,6 +26,11 @@ Options:
     --username NAME         Specifies name of user, mandatory with `--mode client`
 ";
 
+const COPYRIGHT: &str = "natter Copyright (C) 2022 M. Newcomer and K. Siek
```
Comment (kosayoda) [inline src/main.rs:29]: Would this need to include our dear friend AFerrin

Comment (Matthew1231A) [reply src/main.rs:29]: let me add him and redo

**Comment by Matthew1231A** [comment]
Requesting approval.

**Review by kosayoda — APPROVED** [review]
sock rolid

**Pull request merged**